### DISCUSSION TITLE
would it be possible to translate the streak stats?

### DISCUSSION THREAD

**Discussion by c-viniciussantos** [DenverCoder1/github-readme-streak-stats#231] [Ideas]
I would like to translate to pt-br and I can help with the translation

**Comment by DenverCoder1** [comment]
That sounds like a great idea, I should be able to set something up pretty quick to make that possible

**Comment by DenverCoder1** [comment]
@c-viniciussantos 

With #232 it is now possible to add translations :tada: 

Feel free to open a PR updating [`translations.php`](https://github.com/DenverCoder1/github-readme-streak-stats/blob/main/src/translations.php)
